python/paddle/incubate/nn/functional/block_multihead_attention.py

```diff
@@ -30,12 +30,26 @@ def block_multihead_attention(
     block_tables,
     pre_key_cache=None,
     pre_value_cache=None,
+    cache_k_quant_scales=None,
+    cache_v_quant_scales=None,
+    cache_k_dequant_scales=None,
+    cache_v_dequant_scales=None,
+    qkv_out_scale=None,
+    qkv_bias=None,
+    out_shift=None,
+    out_smooth=None,
```

Comment:
The newly added parameters, such as `cache_k_quant_scales`, need to be described in docstring of section `Args` below

Reply:
Normally, the added parameters should be added at the end to ensure compatibility

Reply:
This will be added in next PR.